Starting multilingual post

yabellini opened the pull request:
Starting putting together some ideas for the blog post on multilingual publishing

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+In 2018, the R Community in Latin America collectively translated the [R for Data Science (R4DS) book](https://es.r4ds.hadley.nz/) to Spanish.  This includes the translation of the package [datos](https://github.com/cienciadedatos/datos) to Spanish and dados to [Portuguese](https://cran.r-project.org/web/packages/dados/index.html), becoming an excellent tool for teaching. From the success of the  ...
+
+Driven by this active and growing spanish-speaking community, rOpenSci [successfully piloted](https://ropensci.org/blog/2021/07/27/censo2017/) our first [Spanish-language peer review](https://ropensci.org/commcalls/2019-06-28/), where the submission itself, and all reviews, and editorial responses were in Spanish.
```
```suggestion
Driven by this active and growing Spanish-speaking community, rOpenSci [successfully piloted](https://ropensci.org/blog/2021/07/27/censo2017/) our first [Spanish-language peer review](https://ropensci.org/commcalls/2019-06-28/), where the submission itself, and all reviews, and editorial responses were in Spanish.
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+## Technical materials, technical solutions
+
+rOpenSci provides community support, standards, and infrastructure for scientists and research software engineers working in R to develop, maintain, and publish high-quality open-source scientific software. In addition, we develop and maintain high-quality documentation and resources to support these activities.  
```
```suggestion
rOpenSci provides community support, standards, and infrastructure for scientists and research software engineers working in R to develop, maintain, and publish high-quality open-source scientific software. In addition, we develop and maintain high-quality documentation and [resources](/resources/) to support these activities.  
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+rOpenSci provides community support, standards, and infrastructure for scientists and research software engineers working in R to develop, maintain, and publish high-quality open-source scientific software. In addition, we develop and maintain high-quality documentation and resources to support these activities.  
+
+We publish our material using technical documents which are rendered using software. The translations of technical documents have two aspects (a) the internationalization (i18n) that provides the framework to support translation and requires technical knowledge (need to deal with source code vs. rendered version of the material), and (b) the localization (l10n), which is the task of translating t ...
```
could the part in parenthesis be removed?

Also, in our case, translating the text _also_ require technical knowledge, but on the topic of the content, not on the topic of R Markdown / Quarto /etc.

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+We publish our material using technical documents which are rendered using software. The translations of technical documents have two aspects (a) the internationalization (i18n) that provides the framework to support translation and requires technical knowledge (need to deal with source code vs. rendered version of the material), and (b) the localization (l10n), which is the task of translating t ...
+
+At the same time, technical translations of living documents have two well-defined stages involving different resources:  (a) achieving a first version of the translated material and (b) keeping the material updated and synchronized between the different languages.
```
```suggestion
Besides, technical translations of living documents have two well-defined stages involving different resources:  (a) achieving a first version of the translated material and (b) keeping the material updated and synchronized between the different languages.
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+At the same time, technical translations of living documents have two well-defined stages involving different resources:  (a) achieving a first version of the translated material and (b) keeping the material updated and synchronized between the different languages.
+
+There are several approaches and technologies to work on this task. Based on the experience gain on the mentions previews projects this project aims to advance in stage (a) achieving a first revised translation in Spanish of the rOpenSci materials and testing two different technical workflows to perform the translations.
```
```suggestion
There are several approaches and technologies to work on translations. Based on the experience gain on the mentions previews projects this project aims to advance in stage (a) achieving a first revised translation in Spanish of the rOpenSci materials and testing two different technical workflows to perform the translations.
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+There are several approaches and technologies to work on this task. Based on the experience gain on the mentions previews projects this project aims to advance in stage (a) achieving a first revised translation in Spanish of the rOpenSci materials and testing two different technical workflows to perform the translations.
+
+The first workflow will use GitHub and GitHub review as a platform for translations and revisions since our materials are on this platform, and it has already been used in other translation projects.  
```
but in the end we are testing one workflow only? Which is a combination of both things described here?

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+The second approach will test to generate a workflow using an API of an automatic translator to generate a first translated version of the documents that will be later reviewed by professional bilingual developers also using GitHUb as platform.
+
+The advantage of the second workflow is to reduce the time it takes to produce a quality translation.  Reviewers will focus on linguistic tasks such as the use of inclusive language, the localization of examples, and the correct translation of phrases, metaphors, or analogies. They will also look for references in the language of the translation. In addition, if the workflow proves feasible and u ...
```
```suggestion
The advantage of the second workflow is to reduce the time it takes to produce a quality translation.  Reviewers will focus on linguistic tasks such as the use of inclusive language, the localization of examples, and the correct translation of phrases, metaphors, or analogies. They will also look for references in the language of the translation. In addition, if the workflow proves feasible and useful, it could be incorporated into the pipeline to keep the texts in different languages synchronized, contributing to solve stage (b).
```

Maybe clarify what "references" mean here?

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+The advantage of the second workflow is to reduce the time it takes to produce a quality translation.  Reviewers will focus on linguistic tasks such as the use of inclusive language, the localization of examples, and the correct translation of phrases, metaphors, or analogies. They will also look for references in the language of the translation. In addition, if the workflow proves feasible and u ...
+
+Besides systematizing both experiences, other products of the project will be: (a) a general guide of rules to follow when translating technical documents, (b) a particular guide for the Spanish language, (c) a list of technical terms that are translated and those that are not, (d) an R package with our workflow to translate technical documents. 
```
```suggestion
Besides systematizing both experiences, other products of the project will be: (a) a general guide of rules to follow when translating technical documents, (b) a particular guide for the Spanish language, (c) a list of technical terms that are translated to Spanish and those that are not, (d) an R package with our workflow to translate technical documents. 
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+## Building Community
+
+Translating rOpenSci material to Spanish will help democratize access to these quality resources to learn and teach the necessary digital skills to become contributors to Open Source projects not only because you can now learn how to be a developer, a maintainer, a reviewer or an editor, but also because translation is often a good first way to contribute and is recognized as valuable by the comm ...
```
```suggestion
Translating rOpenSci material to Spanish will help democratize access to these quality resources to learn and teach the necessary digital skills to become contributors to Open Source projects not only because you can now learn how to be a developer, a maintainer, a reviewer or an editor; but also because translation is often a good first way to contribute, and is recognized as valuable by the community. This material is also useful for learning and teaching how to have a workflow to perform computationally reproducible Open Science.
```

maelle reviewed (commented):
Thank you :rocket: :tokyo_tower: (this is the Babel tower of course :innocent: )

yabellini commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+date: '2022-11-29'
+slug: multilingual-publishing-en
+tags:
+  - community
```
mmm...yes!

paocorrales commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,63 @@
+
+In 2017 several Latin American R-Ladies start with the translations to Spanish of the R-Ladies' [Code of Conduct](https://github.com/rladies/starter-kit/wiki/Code-of-Conduct#spanish) and their Rules and Guidelines. 
+
+In 2018, the R Community in Latin America collectively translated the [R for Data Science (R4DS) book](https://es.r4ds.hadley.nz/) to Spanish.  This includes the translation of the package [datos](https://github.com/cienciadedatos/datos) to Spanish and dados to [Portuguese](https://cran.r-project.org/web/packages/dados/index.html), becoming an excellent tool for teaching. From the success of the  ...
```
There isn 't a package in English, the package datos is a collection of data sets spread through different R packages like ggplot2. Maybe we can refrase it as:

```suggestion
In 2018, the R Community in Latin America collectively translated the [R for Data Science (R4DS) book](https://es.r4ds.hadley.nz/) to Spanish.  This includes the translation of all the data sets included in the book compiled in the package [datos](https://github.com/cienciadedatos/datos) in Spanish and dados in [Portuguese](https://cran.r-project.org/web/packages/dados/index.html), becoming an excellent tool for teaching. From the success of the collective and volunteer translation of R4DS, the community also translated [Teaching Tech Together](http://teachtogether.tech/es/index.html) and contributed to several translations to Spanish of [RStudio CheatSheets](https://posit.co/resources/cheatsheets/?type=translations/#translation-12), [The Carpentries](https://software-carpentry.org/lessons/)’ and [The Programming Historian](https://programminghistorian.org/es/) lessons. 
```

paocorrales commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+
+  2. keeping the material updated and synchronized between the different languages.
+
+There are many solutions for internationalizing and localizing content and software. _Translation_ is typically the most time-consuming component of these efforts. [^3]
```
```suggestion
There are many solutions for internationalizing and localizing content and software. _Translation_ is typically the most time-consuming component of these efforts[^3]. 
```

yabellini commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+
+Based on the experience gain on the previews projects and the characteristics of our community, this project aims to advance in stage 1 achieving a first revised translation in Spanish of the rOpenSci materials, and testing a technical workflow to perform the translations.
+
+Our approach will test to generate a workflow using an API of an automatic translator to generate a first translated version of the documents that will be later reviewed by professional bilingual developers using GitHUb as platform.
```
Yes.

eliocamp commented:
I have a PR on top of this PR with my changes. 

https://github.com/ropensci/roweb3/pull/463

yabellini commented:
* [ ] I have read the Content Guidelines.
* [ ] I have read the Technical Guidelines.
* [ ] I used or followed the R Markdown or Markdown template.
* [ ] I have followed the Style Guide.
* [ ] I created or updated my author metadata with correct folder name.
* [ ] I have added relevant tags after browsing existing tags (including "community" tag).
* [ ] I have added the "tech notes" tag if this is a technote.
* [ ] I ran `roblog::ro_lint_md()` on index.md (optional).
* [ ] I ran `roblog::ro_check_urls()` on index.md (optional).
* [ ] I ran a spell-check on index.md.

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+---
+
+We are excited to announce that, with the support of the Chan-Zuckerberg Initiative, NumFOCUS, and the R Consortium, we have begun the process of translating rOpenSci's materials on best practices for software development, code review, and contribution to open source projects into Spanish. As part of this effort, we are also developing guidelines and tools for translating open source resources to ...
```
```suggestion
We are excited to announce that, with the support of the Chan Zuckerberg Initiative, NumFOCUS, and the R Consortium, we have begun the process of translating rOpenSci's materials on best practice for software development, code review, and contribution to open source projects into Spanish. 
As part of this effort, we are also developing guidelines and tools for translating open source resources to reach a wider audience.
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+In 2017, several Latin American R-Ladies began translating the R-Ladies' [Code of Conduct](https://github.com/rladies/starter-kit/wiki/Code-of-Conduct#spanish) and their Rules and Guidelines into Spanish.
+
+In 2018, the R community in Latin America collectively translated the [R for Data Science](https://es.r4ds.hadley.nz/) book into Spanish. This included the translation of all the data sets used in the book, which were compiled in the [datos](https://github.com/cienciadedatos/datos) package, making it an excellent tool for teaching.
```
is there a recording of a talk about this project? I vaguely remember there was one at a LatinR conference but maybe I'm wrong?

yabellini replied:
Yes! It is a talk by Riva on RStudioConf: https://youtu.be/kuHjpdgkSxg
She also talks at useR! 2019 but no recording of that one.

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+## Language representation
+
+The Spanish-speaking R community has been very active and growing in recent years, and has undertaken various translation activities for technical materials such as books, cheat sheets, guides, and datasets.
```
```suggestion
The Spanish-speaking R community has been very active and growing in recent years, and has undertaken various translation activities for technical materials such as books, cheatsheets, guides, and datasets.
```

since that's how Posit writes it

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+In 2018, the R community in Latin America collectively translated the [R for Data Science](https://es.r4ds.hadley.nz/) book into Spanish. This included the translation of all the data sets used in the book, which were compiled in the [datos](https://github.com/cienciadedatos/datos) package, making it an excellent tool for teaching.
+
+The community continued with the translation of [Teaching Tech Together](http://teachtogether.tech/es/index.html) and contributed to the translations into Spanish of the [Posit CheatSheets](https://posit.co/resources/cheatsheets/?type=translations/#translation-12),
```
```suggestion
The community continued with the translation of [Teaching Tech Together](http://teachtogether.tech/es/index.html) and contributed to the translations into Spanish of the [Posit Cheatsheets](https://posit.co/resources/cheatsheets/?type=translations/#translation-12),
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+By publishing multilingual resources, we can lower these barriers by increasing access to knowledge and democratizing access to quality resources, thereby increasing the potential for individuals to contribute to software and open science projects.
+
+## Language representation
```
could some of the paragraphs in this section be merged?

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+* The _internationalization_ refers to the technological solution allowing the software to adapt to different regions without requiring engineering changes to the source code. It is what enables us to localize our content[^3].
+
+* The _localization_ is the process of taking a piece of content in its original form and converting it into something more accessible and suitable for another region, country, or audience [^3]. This includes language, date formats, currency, measurement units, and support for different character sets. 
```
```suggestion
* The _localization_ is the process of taking a piece of content in its original form and converting it into something more accessible and suitable for another region, country, or audience[^3]. This includes language, date formats, currency, measurement units, and support for different character sets. 
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+There are many solutions for internationalizing and localizing content and software. _Translation_ is typically the most time-consuming component of these efforts[^3].
+
+Based on the experience gained from out pilot projects and the characteristics of our community, this project aims to progress to stage 1 by achieving a first revised translation in Spanish of the rOpenSci materials, and testing a technical workflow for performing the translations.
```
```suggestion
Based on the experience gained from our pilot projects and the characteristics of our community, this project aims to progress to stage 1 by achieving a first revised translation in Spanish of the rOpenSci materials, and testing a technical workflow for performing the translations.
```

are these pilot projects or "first experiences"?

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+Based on the experience gained from out pilot projects and the characteristics of our community, this project aims to progress to stage 1 by achieving a first revised translation in Spanish of the rOpenSci materials, and testing a technical workflow for performing the translations.
+
+Our approach will test and develop a workflow using an API of an automatic translator to generate a first translated version of the documents, which will be later reviewed by professional bilingual developers using GitHub as a platform.
```
why not mention DeepL?

maelle reviewed (approved):
:rocket:

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+tags:
+  - community
+  - multilingual
+description: ""
```
don't forget to fill this :-)

yabellini commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,95 @@
+
+In 2017, several Latin American R-Ladies began translating the R-Ladies' [Code of Conduct](https://github.com/rladies/starter-kit/wiki/Code-of-Conduct#spanish) and their Rules and Guidelines into Spanish.
+
+In 2018, the R community in Latin America collectively translated the [R for Data Science](https://es.r4ds.hadley.nz/) book into Spanish. This included the translation of all the data sets used in the book, which were compiled in the [datos](https://github.com/cienciadedatos/datos) package, making it an excellent tool for teaching.
```
```suggestion
In 2018, the [R community in Latin America collectively translated](https://youtu.be/kuHjpdgkSxg) the [R for Data Science](https://es.r4ds.hadley.nz/) book into Spanish. This included the translation of all the data sets used in the book, which were compiled in the [datos](https://github.com/cienciadedatos/datos) package, making it an excellent tool for teaching.
```

yabellini reviewed (commented):
Adding link to Riva's talk

yabellini commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,96 @@
+
+There are many solutions for internationalizing and localizing content and software. _Translation_ is typically the most time-consuming component of these efforts[^3].
+
+Based on the experience gained from our pilot projects and the characteristics of our community, this project aims to progress to stage 1 by achieving a first revised translation in Spanish of the rOpenSci materials, and testing a technical workflow for performing the translations.
```
```suggestion
Based on the experience gained from our first experiences and the characteristics of our community, this project aims to progress to stage 1 by achieving a first revised translation in Spanish of the rOpenSci materials, and testing a technical workflow for performing the translations.
```

yabellini commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,96 @@
+
+Based on the experience gained from our pilot projects and the characteristics of our community, this project aims to progress to stage 1 by achieving a first revised translation in Spanish of the rOpenSci materials, and testing a technical workflow for performing the translations.
+
+Our approach will test and develop a workflow using an API of an automatic translator to generate a first translated version of the documents, which will be later reviewed by professional bilingual developers using GitHub as a platform.
```
```suggestion
Our approach will test and develop a workflow using the automatic translator DeepL API to generate a first translated version of the documents, which will be later reviewed by professional bilingual developers using GitHub as a platform.
```

maelle commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,96 @@
+According to a recent report from The Linux Foundation, _language access_ is one of the _environmental barriers to equity in open source_. The report states that 
+
+> The predominant language of open source is English—in code, content, and community interactions—and English proficiency is a metric by which performance and personality can be judged[^1]. 
+
```
```suggestion

This might remind some readers of the famous [quote](https://www.imdb.com/title/tt4166966/characters/nm0005527) from [Modern Family](https://www.tiktok.com/@primevideouk/video/6899066588121713921), where a character called Gloria exclaims:

> Do you even know how smart I am in Spanish?

```

maelle replied:
@yabellini just to be sure we're not infringing copyrights, I am linking out to it but maybe I'm being overly cautious.

steffilazerte commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+
+* a general guide of rules to follow when translating technical documents,
+* a particular guide for the Spanish language,
+* a list of technical terms that are translated into Spanish and those that are not,
```
```suggestion
* a list of technical terms which will be translated into Spanish and a list of those that will not be,
```

steffilazerte commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+* a list of technical terms that are translated into Spanish and those that are not,
+* an R package with our workflow for translating technical documents.
+
+## Building Community
```
```suggestion
## Building community
```

steffilazerte commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+Multilingual publishing will help democratize access to quality resources for learning and teaching the necessary digital skills to become contributors to open source projects and to have a workflow for performing computationally reproducible open science.
+
+At rOpenSci, we understand review as a way of building community.
+The material for our review process in Spanish will allow rOpenSci to continue building the community in Spanish-speaking regions, such as Latin America, to increase the number of contributors and get feedback on how our tools and processes can be improved to better serve these community members.
```
```suggestion
Developing the material for our review process in Spanish will allow rOpenSci to continue building the community in Spanish-speaking regions (such as Latin America), to increase the number of contributors, and to get feedback on how our tools and processes can be improved to better serve these community members.
```

steffilazerte commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+We are excited to announce that, with the support of the Chan Zuckerberg Initiative, NumFOCUS, and the R Consortium, we have begun the process of translating rOpenSci's materials on best practice for software development, code review, and contribution to open source projects into Spanish. 
+As part of this effort, we are also developing guidelines and tools for translating open source resources to reach a wider audience.
+
+According to a recent report from The Linux Foundation, _language access_ is one of the _environmental barriers to equity in open source_. The report states that 
```
```suggestion
According to a [recent report](https://www.linuxfoundation.org/research/the-2021-linux-foundation-report-on-diversity-equity-and-inclusion-in-open-source) from The Linux Foundation, _language access_ is one of the _environmental barriers to equity in open source_. The report states that 
```

steffilazerte commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+[The Carpentries](https://software-carpentry.org/lessons/)’ 
+and [The Programming Historian](https://programminghistorian.org/es/) lessons.
+
+Driven by this active and growing Spanish-speaking community, rOpenSci [successfully piloted](https://ropensci.org/blog/2021/07/27/censo2017/) our first [Spanish-language peer review](https://ropensci.org/commcalls/2019-06-28/), where the submission itself, as well as all reviews and editorial responses, were in Spanish.
```
```suggestion
Driven by this active and growing Spanish-speaking community, rOpenSci [successfully piloted](/blog/2021/07/27/censo2017/) our first [Spanish-language peer review](/commcalls/2019-06-28/), where the submission itself, as well as all reviews and editorial responses, were in Spanish.
```

steffilazerte commented on content/blog/2022-11-29-multilingual-publishing-en/index.md:
```diff
@@ -0,0 +1,100 @@
+
+In 2018, the [R community in Latin America collectively translated](https://youtu.be/kuHjpdgkSxg) the [R for Data Science](https://es.r4ds.hadley.nz/) book into Spanish. This included the translation of all the data sets used in the book, which were compiled in the [datos](https://github.com/cienciadedatos/datos) package, making it an excellent tool for teaching.
+
+The community continued with the translation of [Teaching Tech Together](http://teachtogether.tech/es/index.html) and contributed to the translations into Spanish of the [Posit Cheatsheets](https://posit.co/resources/cheatsheets/?type=translations/#translation-12),
```
```suggestion
The community continued with the translation of [Teaching Tech Together](https://teachtogether.tech/es/index.html) and contributed to the translations into Spanish of the [Posit Cheatsheets](https://posit.co/resources/cheatsheets/?type=translations/#translation-12),
```

steffilazerte reviewed (commented):
Great post! I love the flow and build up. 

I changed back the folder name because all the comments I made were on that original folder and I couldn't see them on the newly named folder 😱 . So changing the folder name to 2023-01-12 is still necessary (sorry!)

@maelle are you able to take a look at the suggestions for @yabellini? With feedback/approval from @eliocamp and @paocorrales of course! :grin: 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [x] publication date is ok -> **Need to update folder and YAML**
* [X] alternative text of images is informative
* [X] ~~Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")~~
* [x] author metadata is provided with correct folder name -> **Need [folders](https://blogguide.ropensci.org/authortechnical.html#createauthorfile) for @eliocamp and @paocorrales, can the two of you add these?**
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

steffilazerte commented:
Great thanks!
